src/main/java/com/team/planopedia/API/translators/OpenWeatherMapAPI.java

```diff
@@ -0,0 +1,140 @@
+/**
+ * Weather API class Class used to make calls to the WeatherAPI given data from
+ * the WeatherAdapter
+ *
+ * Last updated September 30, 2021
+ * @Author Nimra Sami
+ * @Author Arturo Serdan
+ */
+
+package com.team.planopedia.API.translators;
+import com.team.planopedia.API.interfaces.WeatherAPIInterface;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.json.*;
+
+public class OpenWeatherMapAPI implements WeatherAPIInterface {
+
+    // In order to create a HTTP Connection.
+    private final static String baseURL = "http://api.openweathermap.org";
+    private final static String callAction = "/data/2.5/weather?q=";
+    //Private number of parameters to be returned
+    private final static int numOfParametersReturned = 2;
+
+    /**
+     * Method used to grab data from the weather API, and return the data into
+     * an Array String
+     *
+     * @param _zipCode
+     * @param _country
```

Comment:
Unclear how I should specify the country. For example, how would I specify "United States"? Maybe provide a link to a reference for the API